src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketAuthorMessageTest.java

```diff
@@ -0,0 +1,211 @@
+package com.cloudbees.jenkins.plugins.bitbucket;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.annotation.Nonnull;
```

Comment:
no grouping of imports A-Z non static before static

Reply:
I develop in eclipse with automatic organize import. Are there eclipse format preferences somewhere to develop with Jenkins (like for maven or eclipse organisation)?

Because (if used) this make develop and contribution easier and avoid human checkstyle. Good alternative could be define checkstyle for the maven plugin at build time.